rsciio/tiff/_api.py

```diff
@@ -696,7 +696,11 @@ def _get_hamamatsu_streak_description(tiff, op):
     scaling = dict_meta["Scaling"]
 
     # Address in file where the X axis is saved
-    x_scale_address = int(re.findall(r"\d+", scaling["ScalingXScalingFile"])[0])
+    # If x axis is "Other" it just loads the axis as pixels
```

Comment:
Would it make sense to add comment explaining in what situation is the x axis "Other"?

Reply:
I amended the comment